packages/features/bookings/lib/handleNewBooking/test/team-bookings/collective-scheduling.test.ts

```diff
@@ -1287,7 +1287,7 @@ describe("handleNewBooking", () => {
           expectBookingCreatedWebhookToHaveBeenFired({
             booker,
             organizer,
-            location: OrganizerDefaultConferencingAppType,
+            location: BookingLocations.CalVideo,
```

Comment:
webook responses contains exact location of booking
which is calvideo in this case